cmd/tsconnect/wasm/wasm_js.go

```diff
@@ -380,6 +391,36 @@ func (s *jsSSHSession) Resize(rows, cols int) error {
 	return s.session.WindowChange(rows, cols)
 }
 
+func (i *jsIPN) fetch(url string) js.Value {
+	return makePromise(func() (any, error) {
```

Comment:
404 error

Reply:
This Pull Request was submitted six days ago, comments on it now can't be acted on. If you believe this code is incorrect in some way please feel free to send a pull request to address the concerns.